### PR TITLE
Move metrics-server to fury-kubernetes-monitoring

### PR DESCRIPTION
**Pull request by lnovara**
The aim of this PR is to move `metrics-server` to `fury-kubernetes-monitoring`.

After this PR is merged, we can proceed with merging sighupio/fury-kubernetes-aws#34.

### CODE REVIEW

File: katalog/metrics-server/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  template:
+    metadata:
+      labels:
+        app: metrics-server
+    spec:
+      serviceAccountName: metrics-server
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 1000
+        fsGroup: 2000
```
Comment (ralgozino) [inline katalog/metrics-server/deploy.yml]: Just double-checking here, is it OK that fsgroup != user id?

File: katalog/metrics-server/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+Metrics Server is a cluster-wide aggregator of resource usage metrics for pods
+and nodes.  These are the same metrics that you can access by using `kubectl
+top`. Metrics server collects metrics from the Summary API, exposed by Kubelet
+on each node.
+
+
+## Requirements
+
+- Kubernetes >= `1.8.0`
+- Kustomize >= `1.0.10`
```
Comment (angelbarrera92) [inline katalog/metrics-server/README.md]: v3 please. We have to force using v3

**Comment by angelbarrera92** [comment]
Hi @lnovara, i reviewed the pipeline. I see some tests failing but the pipeline is in green color

http://ci.sighup.io/sighupio/fury-kubernetes-monitoring/655/1/2. Related to the lint job/step

**Comment by lnovara** [comment]
You're right, the lint part is failing. This was an early attempt at applying some policies to our manifests. Do we want to keep them (and fix them as well)? Or should we drop them?

cc @phisco

**Comment by angelbarrera92** [comment]
if it makes sense, i want to have it fixed ;)
Then we can adopt this lint in other projects

**Comment by lnovara** [comment]
The main point here is that I don't know if Python is the right tool for the jobs, we should consider using something like [conftest](https://github.com/instrumenta/conftest) or [kubeval](https://github.com/instrumenta/kubeval).

What do you think?

cc @ralgozino @phisco

File: katalog/metrics-server/deploy.yml (hunk 1 above)
Comment (lnovara) [reply katalog/metrics-server/deploy.yml]: Yes, `fsGroup` is there only to enforce the group ownership of the files written by the container.

**Comment by lnovara** [comment]
@angelbarrera92 since this card was opened (https://trello.com/c/lu7Dphcq/115-define-basic-validation-tests-for-every-manifest), do you think we can remove the Python "tests" from the pipeline?

Furthermore, can you please check the Drone pipeline in this PR just to make sure that everything is correct? 🙏 

Thank you.

**Comment by angelbarrera92** [comment]
> @angelbarrera92 since this card was opened (https://trello.com/c/lu7Dphcq/115-define-basic-validation-tests-for-every-manifest), do you think we can remove the Python "tests" from the pipeline?
> 
> Furthermore, can you please check the Drone pipeline in this PR just to make sure that everything is correct? 🙏
> 
> Thank you.

Hi Luca. Feel free to remove that pipeline part and continue merging it ;)

**Comment by lnovara** [comment]
[Move metrics-server from AWS installer to monitoring module](https://trello.com/c/KFfSaWI3/52-move-metrics-server-from-aws-installer-to-monitoring-module)

**Pull request merged**